<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rewritepwd</class>
 <widget class="QDialog" name="rewritepwd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>639</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>密码重置</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/new/prefix1/0eefae7af46b407ae390a0fad8c36b7e0bb7a043d956e-UVgqj9_fw658.webp);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QPushButton" name="rewritepwd_2">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>确认重置密码</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLineEdit" name="Reuserid">
       <property name="minimumSize">
        <size>
         <width>490</width>
         <height>80</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>请输入学号......</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="sendCodeButton">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>发送验证码</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLineEdit" name="qqemailMa">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>80</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>请输入验证码......</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QLineEdit" name="confirmnewpwd">
       <property name="minimumSize">
        <size>
         <width>490</width>
         <height>80</height>
        </size>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>请确认新密码......</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLineEdit" name="newpwd">
       <property name="minimumSize">
        <size>
         <width>490</width>
         <height>80</height>
        </size>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>请输入新密码......</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="back2Button">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
